src/adapters/node.js

```diff
@@ -1,36 +1,53 @@
-// Generated by CoffeeScript 1.9.3
-(function() {
+(function () {
     var request = require('request');
     var mkFhir = require('../fhir');
     var Q = require('q');
 
     var adapter = {
         defer: Q.defer,
-        http: function(args) {
+        http: function (args) {
             var deff = Q.defer();
-            args.body = args.data;
+
+            if (args.resource != undefined) {
+                args.body = args.resource
+            }
             // url should be relative to baseUrl.
-            if(args.url) {
+            if (args.url) {
                 args.url = args.url.replace(args.baseUrl, '');
+            } 
+            // does not like that for a transaction
+            if (args.url == "") {
```

Comment:
Could this block be eliminated by simply adding a forward slash (rather than "") as the replacement character in the regular expression three lines above? Or would thst approach fail for requests other than transaction?

Reply:
this fails for other transactions like conformance when I add that, the url is then //metadata